checkov/github/base_github_configuration_check.py

```diff
@@ -40,3 +41,14 @@ def resolve_ckv_metadata_conf(conf: dict[str, Any]) -> tuple[dict[str, Any], dic
                 del new_conf[CKV_METADATA]
                 return ckv_metadata, new_conf
         return {}, conf
+
+    def get_result_configuration(
+            self, evaluated_key: str, conf: dict[str, Any]) -> dict[str, Any] | str | list[str | dict[str, Any]]:
+        # if the evaluated key points to a key within an object, the result config should be its parent, for context
+        # example 1 - key: `object.nested_object.key` -> json_path should be `$.object.nested_object`
```

Comment:
is there a case, where even the `nested_object` doesn't exist?

Reply:
If it doesn't exist then 'evaluated_conf' (line 52) would be empty and the function would just return the entire conf.
But maybe it's better in that case to return the `object` is that your suggestion? @gruebel

Reply:
yeah correct, that was my thought 🙂 but probably the whole `conf` is also fine

Reply:
@gruebel implemented your suggestion 👍